src/test/resources/hardware-exploration-chart-template.html

```diff
@@ -80,7 +80,10 @@
                 responsive: true,
                 title: {
                     display: true,
-                    text: '<%= application =%> hardware exploration: error rate'
+                    text: [
+                        '<%= application =%> hardware exploration: max action error rate',
+                        'how broken is the most broken action?'
```

Comment:
I changed the chart title to better explain what it means now. I didn't refresh the screenshots after that.